src/shared/components/oncoprint/DeltaUtils.ts

```diff
@@ -25,6 +25,7 @@ import {
 } from "./TooltipUtils";
 import {MolecularProfile} from "../../api/generated/CBioPortalAPI";
 import {AlterationTypeConstants} from "pages/resultsView/ResultsViewPageStore";
+import ifndef from "../../lib/ifndef";
```

Comment:
if not defined?  hard to know what this is

Reply:
oh sorry, ifndef is a preprocessor command from C that does the same thing, so I thought it would work as a name.

It stands for if not defined